lib/praxis/validation_handler.rb

```diff
@@ -0,0 +1,10 @@
+module Praxis
+  class ValidationHandler
+
+    # Should return the Response to send back
+    def handle!(summary:, errors: nil, exception: nil, **opts)
```

Comment:
I think we need to add an explicit named arg that says where the validation failed...
something like `where:` ? or `failed_in:'? ... or?...

Which could take :reponse (for response validation), request: ...